crates/autopilot/src/run.rs

```diff
@@ -396,58 +394,6 @@ pub async fn run(args: Arguments) {
         args.price_estimation.quote_verification,
     ));
 
-    if let Some(ethflow_contract) = args.ethflow_contract {
```

Comment:
Just moved down to be able to first create the `Maintenance` struct and later add the optional ethflow stuff.